src/main/java/Main.java

```diff
@@ -0,0 +1,23 @@
+import dao.AccountDao;
+import dao.DBCreator;
+
+public class Main {
+
+    public static void main(String[] args) {
+        try {
+            if (args[0].equals("-fileName")) {
+                DBCreator dbCreator = new DBCreator(args[1]);
+                AccountDao accountDao = new AccountDao(dbCreator);
+
+                BankingSystem bankingSystem = new BankingSystem(accountDao);
+                bankingSystem.showMenu();
+            } else {
+                throw new IllegalArgumentException();
```

Comment:
Can you remove the try-catch block? Instead of that:

- Add one more condition for validating allowed args length.
- Move message from lines 18-20 to the constructor of the IllegalArgumentException.

Reply:
done